h5c/vic/src/vic-webapp/e2e/app.po.ts

```diff
@@ -264,7 +272,7 @@ export class VicWebappPage {
     });
   }
 
-  waitForElementToBePresent(el, timeout = this.opsTimeout, selectBy = 'css') {
+  waitForElementToBePresent(el, selectBy, timeout = this.opsTimeout) {
```

Comment:
Since the vast majority of selectBy is 'css' what we can do here is leave that as a default.

waitForElementToBePresent(el, selectBy= 'css', timeout = this.opsTimeout)

that way the only ones that will need to be passed will be the non-default value 'xpath'

so this.waitForElementToBePresent(this.element, 'xpath') will override the default.